### PR TITLE
Add newscrawl tests, and update some testing infrastructure

### PR DESCRIPTION
**Pull request by gregtatum**
These are the commits that didn't have actionable feedback from #420, so should be easier to merge without handling all of the rest of the feedback.

### CODE REVIEW

**Comment by gregtatum** [comment]
Ah, I forgot this is the reason why I rewrote to python. The test fails in CI but not locally.

**Comment by gregtatum** [comment]
I disabled the tests, but plan on re-enabling them when #420 lands. They pass locally but not in CI due to `bc` not being available.

**Review by eu9ene — APPROVED** [review]
Approving, but I added one suggestion for the tests

File: tests/test_data_importer.py [hunk 1]
```diff
@@ -70,15 +70,71 @@ def test_basic_corpus_import(importer, dataset, data_dir):
...
+        env={
+            "COMPRESSION_CMD": COMPRESSION_CMD,
+            "ARTIFACT_EXT": ARTIFACT_EXT,
+            "WGET": os.path.join(CURRENT_FOLDER, "fixtures/wget"),
+            "MOCKED_DOWNLOADS": get_mocked_downloads(),
+        },
+    )
+
+    prefix = data_dir.join(f"artifacts/{dataset}")
+    mono_data = f"{prefix}.ru.{ARTIFACT_EXT}"
```
[the remark this answers is not in the document]
Comment (gregtatum) [reply tests/test_data_importer.py:131]: I'll address this in #420, as this test is mostly setting it up for the changes I wrote there.

[the remark this answers is not in the document]
Comment (gregtatum) [reply tests/test_data_importer.py:131]: Actually, I'll just handle it since I need to rebase to merge anyway.

**Pull request merged**